oraibridge/assetlist.json

```diff
@@ -0,0 +1,274 @@
+      "display": "USDC EVM",
+      "symbol": "USDC",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.svg"
+      },
+      "coingecko_id": "usd-coin",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.svg"
+        }
+      ]
+    },
+    {
+      "description": "AIRI bridge bsc",
+      "denom_units": [
+        {
+          "denom": "oraib0x7e2A35C746F2f7C240B664F1Da4DD100141AE71F",
+          "exponent": 0
+        },
+        {
+          "denom": "AIRI",
+          "exponent": 18
+        }
+      ],
+      "base": "oraib0x7e2A35C746F2f7C240B664F1Da4DD100141AE71F",
+      "name": "Airight BSC",
+      "display": "AIRI BSC",
+      "symbol": "AIRI",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/airi.svg"
+      },
+      "coingecko_id": "airight",
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/airi.svg"
+        }
+      ]
+    },
+    {
+      "description": "USDT bridge bsc",
+      "denom_units": [
+        {
+          "denom": "oraib0x55d398326f99059fF775485246999027B3197955",
+          "exponent": 0
+        },
+        {
+          "denom": "USDT",
+          "exponent": 18
+        }
+      ],
+      "base": "oraib0x55d398326f99059fF775485246999027B3197955",
+      "name": "Tether BSC",
+      "display": "USDT BSC",
+      "symbol": "USDT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/tether.svg"
+      },
+      "coingecko_id": "tether",
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/tether.svg"
+        }
+      ]
+    },
+    {
+      "description": "USDT bridge tron",
+      "denom_units": [
+        {
+          "denom": "trontrx-mainnet0xa614f803B6FD780986A42c78Ec9c7f77e6DeD13C",
+          "exponent": 0
+        },
+        {
+          "denom": "USDT",
+          "exponent": 6
+        }
+      ],
+      "base": "trontrx-mainnet0xa614f803B6FD780986A42c78Ec9c7f77e6DeD13C",
+      "name": "Tether Tron",
+      "display": "USDT Tron",
+      "symbol": "USDT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/tether.svg"
+      },
+      "coingecko_id": "tether",
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/tether.svg"
+        }
+      ]
+    },
+    {
+      "description": "wTRX bridge tron",
```

Comment:
For all these tokens, please define full trace paths, remove the coingecko id, and point to images added under its original chain. Some of these tokens will have to be registered under /_non-cosmos/... , e.g., TRON I think needs to be registered, but something like USDT is already registered.